examples/table_udf/main.go

```diff
@@ -0,0 +1,115 @@
+package main
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"log"
+	"reflect"
+
+	"github.com/marcboeker/go-duckdb"
+)
+
+var db *sql.DB
+
+type tableUDF struct {
+	n     int64
```

Comment:
Maybe we can make the name for `n` more descriptive, as this will be the main example for people when using table functions.